README.md

```diff
@@ -1,634 +1,1855 @@
+build up an identity, the less likely the person is to abandon that identity. So, how hard will it
+be to get an under-collateralized loan?
+
+In TradFi, a borrower isn't considered a good credit risk unless they have at least 5-7 years of
+good credit. This rule is fairly universal despite there being very few laws that mandate this
+length of time. Instead, this rule arose naturally from lenders learning the hard way. 
+
+TradFi isn't a perfect comparison, though, because government issued IDs are hard to burn and
+create. **This does not mean under-collateralized loans require DIDs**; it just means the
+requirements to get an under-collateralized loan in DeFi should be more strict than in TradFi to
+compensate for this difference. How much more strict should the requirements be?
+
+To answer this question, consider an identity farm attack. Mike is malicious and decides to "farm"
+identities: he has these identities lend to each other, he "nutures" them by keeping up with the
+loan payments, and after several years, he "harvests" them to use as burner identities. How
+practical is this attack? *The answer depends entirely on lenders.*
+
+Let's say it took at least 8 years of good credit for a lender to consider offering an
+under-collateralized loan. And let's say lenders only consider borrowers who had to consistently
+deal with compounding interest rates of 15% or more (8 years of credit with non-compounding or
+interest-free loans would be a red-flag). So, to trick Alice into offering him an
+under-collateralized loan, Mike would need to have his burner identity pay back 8 years worth of
+loans with at least a 15% compounding interest (compounding annually to keep it simple). If he gave
+his burner identity a single 8 year loan for 1000 ADA, it would require 1700 ADA in total to make all
+of the payments ([calculator](https://www.nerdwallet.com/calculator/loan-calculator)). The
+additional 700 ADA is due to the interest. This 1700 ADA is Mike's investment for an 8 year credit
+history of 1000 ADA.
+
+Mike needs to ask Alice for a loan of about 1000 ADA. It would be a red-flag if he suddenly asked
+for 10,000 ADA despite his history only dealing with loans 10x smaller than that. Even in TradFi,
+credit limits are slowly increased as a borrower's credit history matures. Let's be generous and say
+Alice agrees to a 1000 ADA loan where only 80% of the loan's value is backed by tangible assets.
+Therefore, if Mike walked away with the 1000 ADA, he would net 200 ADA. This 200 ADA is his return.
+In otherwords, his 1700 ADA investment yielded a total of 200 ADA over an 8 year time period. That
+equates to an average yearly return of 1.4% (`200 profit / 1700 investment / 8 years * 100`). Mike
+put in all that time and effort farming the identities for only a 1.4% APY. But remember, Mike did
+*not* get 1.4% APY each year for 8 years, he had to wait the full 8 years to see any return from
+this attack.
```

Comment:
Except it doesn't actually cost Mike anything because he is making the payments to himself through his multiple fake identities.  The cost to him is time and transaction fees.

Still I am coming around to agree that under collateralised loans will be made because lenders can check addresses of previous lenders and compare with other lenders they have verified.  This way they could be more certain that the previous loans were real.

Reply:
> Except it doesn't actually cost Mike anything because he is making the payments to himself through his multiple fake identities. The cost to him is time and transaction fees.

I agree there is barely any cost, but that is why I was trying to frame it as invested capital vs return. A loan for 1000 ADA still requires the 1000 ADA. However, perhaps the velocity of the 1000 ADA could be high enough to keep the invested capital fixed :thinking:. If it is possible to use the same 1000 ADA to maintain loans for 10,000 ADA loans, this would make this attack much more feasible.

I may remove this section entirely. I still believe it is possible to have a healthy under-collateralized loan market, but I now don't think my current thinking is correct. Cross-referencing other lenders like you said, or linking borrower IDs to public identities (even without DIDs) may be enough.

Reply:
I removed the under-collateralized loan discussion section and elaborated on the previous [section](https://github.com/fallen-icarus/cardano-loans/tree/v1.0.0.0rc?tab=readme-ov-file#negotiable-collateralization). I just focused on the data analyses one could use to identify fake identities, such as check past lenders.